apps/server/src/utils/__tests__/parser.test.ts

```diff
@@ -550,7 +550,7 @@ describe('test aliases import', () => {
           pathAndParams: 'testpathAndParams',
         },
       ],
-    } as DatabaseModel;
+    } as unknown as DatabaseModel;
```

Comment:
I am not sure the downcasting is a good approach, at this point, the type is not doing anything for us anymore?

Reply:
it is to avoid filling out a full dataset for a test that doesn't need it right?
In that case I think it is ok to force the types to clear out some of the error messages